code/src/qik_2s12v10.cpp

```diff
@@ -3,25 +3,12 @@
 namespace r2d2::moving_platform {
 
 
-    qik_2s12v10_c::qik_2s12v10_c(r2d2::uart_ports_c &uart_port,
```

Comment:
same as before is there a reason why this is on the same line followed by a few arguments on different lines?

Reply:
Your commit has fixed this